app/src/main/java/com/example/admin/personallibrarycatalogue/AddBookActivityFragment.java

```diff
@@ -143,7 +167,7 @@ public void onClick(View v) {
             }
         });
 
-        // Actions for cancel button (clear fields and leave activity)
+        // Actions for cancel button (just leave activity)
```

Comment:
Word cancel already means leaving something without changes. Comment can be removed as redundant